src/main/java/org/springframework/data/elasticsearch/client/elc/IndicesTemplate.java

```diff
@@ -241,6 +248,45 @@ public Map<String, Object> getMapping() {
 		return responseConverter.indicesGetMapping(getMappingResponse, indexCoordinates);
 	}
 
+	@Override
+	public ClusterMapping getClusterMapping() {
+		GetMappingRequest getMappingRequest = requestConverter.indicesGetMappingRequest(IndexCoordinates.of("*"));
```

Comment:
the user might not have the permission to read that information from all indices in a cluster